windows/src/desktop/kmshell/main/OnlineUpdateCheck.pas

```diff
@@ -581,12 +498,8 @@ function TOnlineUpdateCheck.DoRun: TOnlineUpdateCheckResult;
           end
           else if (Length(FParams.Packages) > 0) or (FParams.Keyman.DownloadURL <> '') then
           begin
-            if not FSilent then
-              ShowUpdateForm
-            else
-            begin
-              ShowUpdateIcon;
-            end;
+            // No longer showing user notification through the online update
+            // forms or icon after background windows update see #10038
```

Comment:
I find this comment a bit confusing. Does it compare the current behavior with to the old behavior (aka in earlier versions we showed notifications but we no longer do that), or the behavior in the current update workflow (aka now we hide the notifications)?

Reply:
Yes. Ok I am looking to remove this particular OnlineUpdateCheck unit altogether but there is a HttpServer.App.OnlineUpdate.pas still using it. For the new background update in the design we have removed the extra update icon that was added the system tray as well as the ShowUpdateForm it is now replaced by a tab in the configuration. The comment really doesn't need to be there and looking today I can actually remove the whole if block.